src/main/java/ecdar/presentations/ComponentPresentation.java

```diff
@@ -252,15 +251,15 @@ ModelController getModelController() {
     @Override
     double getDragAnchorMinHeight() {
         final Component component = controller.getComponent();
-        double minHeight = 10 * GRID_SIZE;
+        double minHeight = 100;
```

Comment:
Consider making a function of `Ecdar.CANVAS_PADDING`

Reply:
Changing the padding would have, IMO, unexpected side effect here